packages/core/tests/integration/components/navi-visualization-config/goal-gauge-test.ts

```diff
@@ -0,0 +1,98 @@
+import { run } from '@ember/runloop';
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, fillIn, blur, findAll } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { TestContext as Context } from 'ember-test-helpers';
+import StoreService from '@ember-data/store';
+import NaviVisualizationConfigGoalGaugeComponent from 'navi-core/components/navi-visualization-config/goal-gauge';
+
+const Template = hbs`
+  <NaviVisualizationConfig::GoalGauge
+    @response={{this.response}}
+    @request={{this.request}}
+    @options={{this.options}}
+    @onUpdateConfig={{this.onUpdateConfig}}
+  />`;
+
+type ComponentArgs = NaviVisualizationConfigGoalGaugeComponent['args'];
+
+interface TestContext extends Context, ComponentArgs {}
+
+module('Integration | Component | visualization config/goal gauge', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function(this: TestContext) {
+    const store = this.owner.lookup('service:store') as StoreService;
+    this.options = {
+      baselineValue: 200,
+      goalValue: 0,
+      metricCid: 'cid_bubbles'
+    };
+    this.onUpdateConfig = () => null;
+    this.request = store.createFragment('bard-request-v2/request', {
+      columns: [
+        {
+          cid: 'cid_bubbles',
+          type: 'metric',
+          field: 'bubbles',
```

Comment:
Is this a real field in `bardOne`?

Reply:
Not that Im aware of, but in most of the tests we dont use the columnMetadata so I didnt update them to be something that exists.